test/models/data_export_test.rb

```diff
@@ -5,13 +5,36 @@
 class DataExportTest < ActiveSupport::TestCase
   def setup
     @export1 = data_exports(:data_export_one)
```

Comment:
For all the failed validations below can you update to check the status of the export to ensure that they are set to `error`

Reply:
Same comment as above